src/utils/events/move-input.js

```diff
@@ -24,9 +26,29 @@ export default class MoveInput {
   }
 
   handler(event) {
-    this.callback({
-      srcEvent: event,
-      target: this.element
-    });
+    switch (event.type) {
+    case 'mousedown':
+      if (event.button === 0) {
+        this.pressed = true;
+      }
+      break;
+    case 'mousemove':
+      if (event.which !== 1) {
```

Comment:
`button` only applies to events in which a button is pressed or released:
https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/button. And `which` is non-standard but better supported than `buttons`...IOW this part of JS is a mess that dates back to before there were many "good parts" :/